src/pseudopeople/noise_functions.py

```diff
@@ -353,7 +353,7 @@ def keyboard_corrupt(truth, corrupted_pr, addl_pr, rng):
                 if random_number < corrupted_pr:
                     err += rng.choice(qwerty_errors[token])
                     random_number = rng.uniform()
-                    if random_number < addl_pr:
+                    if random_number > addl_pr:
```

Comment:
I think technically it should be >= if it's really the reverse log, though this has always kinda confused me.

Reply:
I believe yo uare right.

Reply:
It's practically speaking the same thing due to the fact that we're drawing a random float.